### ISSUE TITLE
Make `.toHaveValue()` check value of `aria-valuenow` as a fallback to `value`

### ISSUE THREAD

**Issue by idanen** [testing-library/jest-dom#478]
### Describe the feature you'd like:

Suggesting that `.toHaveValue()` support elements' [`aria-valuenow`](https://developer.mozilla.org/en-US/docs/Web/Accessibility/ARIA/Attributes/aria-valuenow) attribute

### Suggested implementation:

When element doesn't have the value property, fallback to try and get the `aria-valuenow` attribute
